src/applications/vaos/new-appointment/components/DateTimeRequestPage/SelectedIndicator.jsx

```diff
@@ -3,12 +3,13 @@ import moment from 'moment';
 
 export function getSelectedLabel(date, selectedDates) {
   const matchingTimes = selectedDates.filter(selected =>
-    selected.startsWith(date),
+    selected?.startsWith(date),
```

Comment:
I don't think you get this far if selectedDates is empty. So no need for the '?'.

Reply:
Done. Removed the '?'